content/en/docs/components/pipelines/v2/compile-a-pipeline.md

```diff
@@ -5,13 +5,13 @@ weight = 5
 +++
 
 You can compile your pipeline or component to intermediate representation (IR) YAML.  The IR YAML definition preserves a static representation of the pipeline or component. You can submit the YAML definition to the KFP backend for execution, or deserialize it using the KFP SDK for integration into another pipeline.
-([View an example on Github][compiled-output-example]).
+([View an example on GitHub][compiled-output-example]).
 
 **Note:** Pipelines as well as components are authored in Python. A pipeline is a template representing a multistep workflow, whereas a component is a template representing a single step workflow.
 
 ## Compile a pipeline
 
-Pipelines as well as components are workflow templates authored in Python. A pipeline represents a multistep workflow, whereas a component represents a single step workflow. You can compile a pipeline or a template into IR YAML using the `Compiler.compile` method. To do this, follow these steps:
+You can compile a pipeline or a template into IR YAML using the `Compiler.compile` method. To do this, follow these steps:
```

Comment:
Not your code: noticed in the example below that the code is under-indented (two spaces v four)